### DISCUSSION TITLE
Create a new session API

### DISCUSSION THREAD

**Discussion by santhosh-stitch** [zitadel/zitadel#6384] [Q&A]
Hi,

please help us on the following issue we are using zitadel  Create a new session API details of the api call as follows,

api - v2alpha/sessions
method - post
payload : {
"checks": {
"user": {
"loginName": "our user email address"
}
}
}
response : {
    "code": 16,
    "message": "auth header missing"
}

But it requires authorization token. Such an api should not require authorization in Zitadel docs.
why a token is needed for this API. If it neccessary which token need to pass in it.

**Comment by mffap** [comment]
Hi @santhosh-stitch 
You're right, the docs are missing the Auth Header. We will update that, thanks for bringing it up.
You need a valid token from an authorized service user to access the APIs. That could be a PAT or an access token.

**Reply by santhosh-stitch** [reply]
Hi @mffap,

We are using PAT token for session apis but it getting blow response.
response: {
    "code": 7,
    "message": "Errors.OriginNotAllowed (AUTH-DZG21)",
    "details": [
        {
            "@type": "type.googleapis.com/zitadel.v1.ErrorDetail",
            "id": "AUTH-DZG21",
            "message": "Errors.OriginNotAllowed"
        }
    ]
}

But we passing the access_token the aforementioned api will work.
Could you please explain why this is happening?

**Reply by fforootd** [reply]
FYI this is currently discussed in https://discord.com/channels/927474939156643850/1140852225221070968
